scripts/compute_branch_network.sh

```diff
@@ -12,6 +12,9 @@ fi
 if [ "${BRANCH}" = "rel/stable" ]; then
     echo "testnet"
     exit 0
+elif [ "${BRANCH}" = "rel/beta" ]; then
```

Comment:
oh, this makes our branch parent stuff unnecessary doesn't it?

Reply:
Not quite. This catches if rel/beta is passed as an argument, or if `./scripts/compute_branch.sh` actually returns rel/beta. If you do a `checkout -f`, then BRANCH is HEAD, and BRANCHPARENT will be the relevant parent. Hopefully.